### DISCUSSION TITLE
[委托] 是绳匠就来砍气球

### DISCUSSION THREAD

**Discussion by Jem-isme** [share121/inter-knot#99] [其他]
<img width="988" alt="Screenshot 2024-08-09 at 17 36 31" src="https://github.com/user-attachments/assets/84cf4275-94f8-4a10-b661-40b79a1cc5b1">
发现一个特殊空洞!  观察到里面有黄金邦布伪装成气球的样子

**Comment by Modog1145** [comment]
这个委托我接不了！

**Comment by wushanef** [comment]
这个委托我接了
